### PR TITLE
[UOT-145550] Updates to Organization Details page to incorporate `Org Head` and `Org Type(s)` information in table

### PR DESCRIPTION
**Pull request by mishoe**
**Description:**
This PR adds in additional attributes/relationships data into the Organization Details page based on additions to the graph (namely the Organization's Head and Organization Types information). This branch has been tested locally against both a local version of neo4j graph as well as the dev neo4j graph.

A screenshot of the updates (the table on the left hand side has been updated to add `Org Head` and `Org Type(s)`. 
Screen Shot 2022-05-19 at 10 36 58 AM

### CODE REVIEW

File: frontend/src/containers/GameChangerDetailsPage.js [hunk 1]
```diff
@@ -262,6 +268,43 @@ const GameChangerDetailsPage = (props) => {
...
+	const getTypeData = async (name, cloneName) => {
+		let types = '';
+		const resp = await gameChangerAPI.callGraphFunction({
+			functionName: 'getTypeDataDetailsPage',
+			cloneName: cloneName,
+			options: {
+				typeName: name,
+			},
+		});
+		if (resp.data.typeData.nodes && resp.data.typeData.nodes.length > 0) {
```
[the remark this answers is not in the document]
Comment (JLyons1985) [reply frontend/src/containers/GameChangerDetailsPage.js:295]: Agree with Jared here adding in resp.data?.typeData?.nodes should do the trick

**Review by JLyons1985 — CHANGES_REQUESTED** [review]
Add a couple checks in for the resp.data stuff just in case you dont get anything back.

File: frontend/src/containers/GameChangerDetailsPage.js (hunk 1 above)
Comment (britton-jason) [inline frontend/src/containers/GameChangerDetailsPage.js:295]: Whenever parsing arrays, array functions should be used in place of raw for loops https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Optional chaining (`?.` syntax) allows for null/undefined checking in-line. The entire expression will return undefined if any of the checked properties are nullish https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

combine this with another undefined check using logical OR and you get this whole section in 1 short line of code 🙂

`const types = resp?.data?.typeData?.nodes?.join(', ') || '';`

**Review by britton-jason — DISMISSED** [review]

**Comment by mishoe** [comment]
@JLyons1985 @britton-jason @jram930 Code updated with missing data checks, and also consolidated code a bit, everything runs good on my end as well
Screen Shot 2022-05-24 at 12 15 49 PM

Requesting re-review

**Pull request merged**